test/common.js

```diff
@@ -88,6 +88,11 @@ exports.validate = (t, report, options) => {
       t.match(nodeReportSection,
               new RegExp('NodeReport version: ' + nodereportMetadata.version),
               'NodeReport section contains expected NodeReport version');
+      const sysInfoSection = getSection(reportContents, 'System Information');
+      // Find a line which ends with /api.node or \api.node (Unix or Windows paths.)
+      // to see if the library for node report was loaded.
+      t.match(sysInfoSection, /  .*[\/|\\]api\.node/,
```

Comment:
`|` shouldn't be here (unless you meant to allow `|api.node`. Strictly speaking `/` in a character class doesn't need to be escaped so this could be `[/\\]`.